src/Common/ErrorCodes.cpp

```diff
@@ -612,6 +612,8 @@
     M(641, CANNOT_APPEND_TO_FILE) \
     M(642, CANNOT_PACK_ARCHIVE) \
     M(643, CANNOT_UNPACK_ARCHIVE) \
+    M(644, CANNOT_SYSCTL) \
+    M(645, KERNEL_STRUCTURE_SIZE_MISMATCH) \
```

Comment:
I don't think we need to introduce new error codes for one specific place in code, let's just use `SYSTEM_ERROR`
```suggestion
```